### PR TITLE
[FEATURE] Development and testing web server

### PR DESCRIPTION
**Pull request by oliverklee**
The server can be started using this command:

bin/console server:run

Also document the use of the different environments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,37 @@ this code.
...
+
+The server will then listen on `http://127.0.0.1:8000` (or, if port 8000 is
+already in use, on the next free port after 8000).
+
+You can stop the server with CTRL + C.
+
+### Testing
+
+To run the server in testing mode (which normally will only be needed for the
+automated tests, provide the `--dev` option:
```
Comment (samtuke) [inline README.md]: Should this read: "provide the `--env` option" ?

Comment (oliverklee) [inline README.md]: Oh, right. Good catch!

Comment (oliverklee) [inline README.md]: Fixed (and also the other two typos).

**Comment by oliverklee** [comment]
Repushed.

**Pull request merged**